lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -21,8 +21,35 @@ def get_url(self, tool):
         return request.build_absolute_uri(url)
 
 
+class CourseBlockSerializer(serializers.Serializer):
+    """
+    Serializer for Course Block Objects
+    """
+    blocks = serializers.SerializerMethodField()
+
+    def get_blocks(self, blocks):
+        data = {}
+
+        for block_key in blocks:
+            block_data = {
+                'id': str(block_key),
+                'type': str(block_key.category),
+                'display_name': blocks.get_xblock_field(block_key, 'display_name', str(block_key.category)),
+                'lms_web_url': reverse(
```

Comment:
I know you are following convention with this name. But like... Shouldn't this just be `url`?

Reply:
Yeah I kept this as `lms_web_url` because it would impact our use of `normalizeBlocks` in MFE land. On that note, should we be writing our own `normalizeBlocks` for course home? I just reused courseware's existing function because it fit our use case.

Reply:
I would probably say no. If it fits our use case, I think it is beneficial for us to be able to reuse code across courseware/course home. But if our needs start to diverge here, that's when I'd say we could look into a separate function for us.